### PR TITLE
Show loading icon when get request for time series is pending

### PR DESCRIPTION
**Pull request by hvangeffen**

### CODE REVIEW

File: src/components/charts/LoadingOverlay.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    right?: number
+    bottom?: number
+    left?: number
+  }
+}
+
+const props = defineProps<Props>()
+
+const style = computed(() => ({
+  paddingTop: props.margin?.top ? `${props.margin.top}px` : undefined,
```
Comment (ceesvoesenek) [inline src/components/charts/LoadingOverlay.vue:22]: It looks a bit weird that you're setting the padding with a property called `margin`. Maybe add a comment why this is the case?

File: src/components/charts/LoadingOverlay.vue [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="loading-overlay" :style="style">
+    <v-icon size="x-large">mdi-loading mdi-spin</v-icon>
```
Comment (ceesvoesenek) [inline src/components/charts/LoadingOverlay.vue:3]: There is also `<v-progress-circular indeterminate />` that we already use as a loading icon for the WMS layers, and which has a slightly cooler animation 😄

File: src/components/charts/TimeSeriesChart.vue [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="chart-with-chips">
+    <LoadingOverlay v-if="isLoading" class="chart-overlay" :margin="margin" />
```
Comment (ceesvoesenek) [inline src/components/charts/TimeSeriesChart.vue:3]: This class is no longer used?

File: src/services/useTimeSeries/index.ts [hunk 4]
```diff
@@ -21,6 +21,7 @@ export interface UseTimeSeriesReturn {
   series: Ref<Record<string, Series>>
   isReady: Ref<boolean>
   isLoading: Ref<boolean>
+  loadingRequests: Ref<string[]>
```
Comment (ceesvoesenek) [inline src/services/useTimeSeries/index.ts:24]: Maybe `loadingSeriesIds` or something? We have quite a lot of different IDs floating around so it's nice to know which one this array contains.

**Review by ceesvoesenek — COMMENTED** [review]
Looks nice! Some minor comments

File: src/components/charts/LoadingOverlay.vue (hunk 2 above)
Comment (wkramer) [reply src/components/charts/LoadingOverlay.vue]: Discussed with Tom. Use skeleton loader.

**Pull request merged**